src/components/Spinner.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
```

Comment:
the spinner has already been implemented, please remove this and use the one that already exists